src/storm-pars/modelchecker/region/RegionRefinementChecker.cpp

```diff
@@ -10,6 +11,7 @@
 
 #include "storm/exceptions/InvalidArgumentException.h"
 #include "storm/exceptions/NotSupportedException.h"
+#include "utility/logging.h"
```

Comment:
```suggestion
#include "storm/utility/macros.h"
```
to support the STORM_LOG_ASSERT suggested below